modules/nw-multinetwork-sriov.adoc

```diff
@@ -29,6 +29,19 @@ scheduled to worker nodes that have sufficient resources available.
 * The SR-IOV CNI plug-in plumbs VF interfaces allocated from the SR-IOV device
 plug-in directly into a Pod.
 
+You can use the {product-title} console to install SR-IOV, by deploying,
+the SR-IOV Network Operator.  The SR-IOV Network Operator
+creates and manages the components of the SR-IOV stack end-to-end. The operator
+provides following features:
+
```

Comment:
Another feature is to deploy all the necessary components of SR-IOV admission controller. 
And add a note that it's enabled by default, cannot be disabled. because it may affect how user write SR-IOV pod spec.

Reply:
Fixed.